packages/core/component-doc.json

```diff
@@ -2,8 +2,8 @@
   "timestamp": "",
   "compiler": {
     "name": "@stencil/core",
-    "version": "4.13.0",
-    "typescriptVersion": "5.4.2"
```

Comment:
Don't downgrade component-doc.json. Please execute `pnpm install` to install latest dev dependencies.